src/azure-cli/requirements.py3.windows.txt

```diff
@@ -33,7 +33,7 @@ azure-mgmt-containerregistry==3.0.0rc15
 azure-mgmt-containerservice==9.4.0
 azure-mgmt-core==1.2.0
 azure-core==1.7.0
-azure-mgmt-cosmosdb==1.0.0
+azure-mgmt-cosmosdb==2.0.0rc1
```

Comment:
actually rc version is not recommended. is it possible to use stable version?

Reply:
This PR is not to merge here in the public repo, but we have the RBAC operations in this version of the SDK